src/classes/CSVReader.cls

```diff
@@ -0,0 +1,216 @@
+/**
+* File Name :   CSVReader.cls
+* Description : Class to facilitate reading the CSV, if needed in chunks
+*               Modified version of 
+*               https://docs.google.com/leaf?id=0B2snsA4L4ojWNWE0ZGRiMzgtNjA4ZC00OWQwLWFjM2MtMTYzZjI5NmE2YjU5&hl=en&authkey=CLG85PoD
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 01/04/2014 Klea Kolaric   Initial version
+*
+*
+*/
+public with sharing class CSVReader {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.